content/docs/command-reference/commit.md

```diff
@@ -1,8 +1,8 @@
 # commit
 
-Record changes to DVC-tracked files in the <abbr>project</abbr>, by updating
-[DVC-files](/doc/user-guide/dvc-files-and-directories) and saving
-<abbr>outputs</abbr> to the <abbr>cache</abbr>.
+Record changes to DVC-tracked files in the <abbr>project</abbr>, by updating the
+`dvc.lock` or `.dvc` files and saving <abbr>outputs</abbr> to the
+<abbr>cache</abbr>.
```

Comment:
OK this is not incorrect but I feel like we could simplify a little more? Since it's the cmd ref intro. For example "DVC-tracked" files basically means the same as "outputs" — probably we don't need both terms here, they can be mentioned in the longer description later. Check other cmd refs for inspiration 🙂